templates/GeoIp.conf.j2

```diff
@@ -0,0 +1,42 @@
+# {{ ansible_managed }}
+
+# Please see https://dev.maxmind.com/geoip/geoipupdate/ for instructions
+# on setting up geoipupdate, including information on how to download a
+# pre-filled GeoIP.conf file.
+
+# Enter your account ID and license key below. These are available from
+# https://www.maxmind.com/en/my_license_key. If you are only using free
+# GeoLite databases, do not uncomment these lines.
+# TODO - support paid accounts
```

Comment:
Is this `TODO` from you or MaxMind? Do they not support paid accounts at this time?

Reply:
Nope, it's mine. I just didn't want to bother with the conditionals since we don't need them at the moment.

Reply:
Works for me.